static/panes/tool.js

```diff
@@ -495,7 +495,8 @@ Tool.prototype.onCompileResult = function (id, compiler, result) {
                         } else {
                             this.add(
                                 this.clickableUrls(this.normalAnsiToHtml.toHtml(obj.text)),
-                                obj.tag ? obj.tag.line : obj.line
+                                obj.tag ? obj.tag.line : obj.line,
+                                obj.tag ? obj.tag.flow : null
```

Comment:
Just a heads-up that this and #3843 will conflict here. Will need to make sure that some order is followed

Reply:
I think it would be better to merge #3843 first, to have the column first.

Reply:
conflict resolved